src/components/nav/CustomDrawer.tsx

```diff
@@ -4,8 +4,7 @@ import { ThemeContext } from '@src/context/Theme'
 import { highlight as hi, light } from '@styles'
 import { skipRoute } from '@util'
 import { useContext } from 'react'
-import { Image, StyleSheet, Text, View } from 'react-native'
-import { TouchableOpacity } from 'react-native-gesture-handler'
+import { Image, StyleSheet, Text, TouchableOpacity, View } from 'react-native'
 
 import { version } from '../../../package.json'
 // import { useState } from 'react'
```

Comment:
Why?

Reply:
So you need an commented import for that?

Reply:
So the import is needed?
If not could you pls Remove it

Reply:
sure i can remove it